File: Tugas coding modul 5/coding c/PRAK505-2310817120005-Noviana_Nur_Aisyah.c
#include <stdio.h>

void Biodata(int a, char b[], char c[]){
    int tahun_sekarang = 2023;
    int sekarang = 2023 - a;

    printf("\nOutput \n");
    printf("Perkenalkan Nama Saya: %s\n", b);
    printf("Umur Saya: %d\n", sekarang);
    printf("Saya Adalah Angkatan: %d\n", tahun_sekarang);
    printf("Asal Saya dari: %s\n", c);
}
int main() {
    int tahunLahir;
    char Namaku[20], Asal[15];
    scanf(" %d", &tahunLahir);   
    scanf(" %[^\n]%*c", &Namaku);
    scanf(" %[^\n]%*c", &Asal);
    Biodata(tahunLahir, Namaku, Asal);

    return 0;
}

File: Tugas coding modul 5/coding py/PRAK505-2310817120005-Noviana_Nur_Aisyah.py
def Biodata(tahunLahir, Namaku, Asal):
    tahun_sekarang = 2023
    sekarang = 2023 - tahunLahir

    print("\nOutput:")
    print(f"Perkenalkan Nama Saya: {Namaku}")
    print(f"Umur Saya: {sekarang}")
    print(f"Saya Adalah Angkatan: {tahun_sekarang}")
    print(f"Asal Saya dari: {Asal}")

def main():
    tahunLahir = int(input())
    Namaku = input()
    Asal = input()
    Biodata(tahunLahir, Namaku, Asal)

main()